qe/drivers.py

```diff
@@ -2014,29 +1977,23 @@ def implicit_1e_matrix_product(self, V_k):
         try:
             k = V_k.shape[1]
         except IndexError:  # Handle case when V is a vector
+            V_k = V_k.reshape(len(V_k), 1)
             k = 1
```

Comment:
Far better! 
Maybe the best is
```
if V_k.ndim == 1
    V_k = V_k.reshape(len(V_k), 1)

k = V_k.shape[1]
```
```

Reply:
Ah yes! Thanks, I didn't know of this `ndim` command